css-structure/css-structure_custom-properties/css/styles.css

```diff
@@ -4,32 +4,40 @@
   box-sizing: border-box;
 }
 
+:root {
+  --color-black-light: #303030;
+  --color-primary: #8236ffff;
+  --color-white: #ffffff;
+  --letter-spacing-small: 0, 025em;
+  --border-radius-same: 4px;
```

Comment:
This is more of a recommendation.

Perhaps you could choose a more meaningful name, because `-same` doesn't tell you what it stands for. Here some alternatives:

+ `--border-radius-base`
+ `--border-radius-default`
+ `--border-radius-small` etc.